### PR TITLE
fix(dedicated-ip): resolve state and view issue

### PR DESCRIPTION
**Pull request by sachinrameshn**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/ip-strat`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #.MANAGER-10191
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.routes.js [hunk 1]
```diff
@@ -2,11 +2,15 @@ import template from './ip-ip-agoraOrder.html';
...
     url: '/agoraOrder?{catalogName:string}',
-    template,
-    controller,
-    controllerAs: '$ctrl',
+    views: {
+      ipview: {
+        template,
+        controller,
+        controllerAs: '$ctrl',
+      },
```
Comment (mohammed-zahaf) [inline packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.routes.js:12]: why not using component ?

File: packages/manager/apps/dedicated/client/app/ip/ip/ip-ip.html [hunk 2]
```diff
@@ -1,247 +1,179 @@
...
+                <p data-ng-if="alerts.spam.length">
+                    <span
+                        class="d-block"
+                        data-ng-pluralize
+                        data-count="alerts.spam.length"
+                        data-when="{
+                                '0': ('ip_alerts_spam' | translate),
+                                'one': ('ip_alerts_spam_1' | translate),
+                                'other': ('ip_alerts_spam_other' | translate: { count: {} })
+                            }"
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/ip/ip/ip-ip.html:48]: **nitpick:** Can we replace `<span>` + `.d-block` with the block level elements?

File: packages/manager/apps/dedicated/client/app/ip/ip/ip-ip.html [hunk 3]
```diff
@@ -1,247 +1,179 @@
...
+                        class="d-block"
+                        data-ng-pluralize
+                        data-count="alerts.antihack.length"
+                        data-when="{
+                                '0': ('ip_alerts_antihack' | translate),
+                                'one': ('ip_alerts_antihack_1' | translate),
+                                'other': ('ip_alerts_antihack_other' | translate: { count: {} })
+                            }"
+                    >
+                    </span>
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/ip/ip/ip-ip.html:72]: **nitpick:** Can we replace `<span>` + `.d-block` with the block level elements?

File: packages/manager/apps/dedicated/client/app/ip/ip/ip-ip.html [hunk 4]
```diff
@@ -1,247 +1,179 @@
...
+                        class="d-block"
+                        data-ng-pluralize
+                        data-count="alerts.arp.length"
+                        data-when="{
+                                '0': ('ip_alerts_arp' | translate),
+                                'one': ('ip_alerts_arp_1' | translate),
+                                'other': ('ip_alerts_arp_other' | translate: { count: {} })
+                            }"
+                    >
+                    </span>
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/ip/ip/ip-ip.html:94]: **nitpick:** Can we replace `<span>` + `.d-block` with the block level elements?

File: packages/manager/apps/dedicated/client/app/ip/ip/ip-ip.html [hunk 5]
```diff
@@ -1,247 +1,179 @@
...
+                    <span
+                        class="d-block"
+                        data-ng-pluralize
+                        data-count="alerts.mitigation.length"
+                        data-when="{
+                                '0': ('ip_alerts_mitigation' | translate),
+                                'one': ('ip_alerts_mitigation_1' | translate),
+                                'other': ('ip_alerts_mitigation_other' | translate: { count: {} })
+                            }"
                     >
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/ip/ip/ip-ip.html:115]: **nitpick:** Can we replace `<span>` + `.d-block` with the block level elements?

File: packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.routes.js (hunk 1 above)
Comment (sachinrameshn) [reply packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.routes.js:12]: HI Mohammed, Yes we can do it. Can we do it as part of refactor ?

File: packages/manager/apps/dedicated/client/app/ip/ip/ip-ip.html (hunk 3 above)
Comment (sachinrameshn) [reply packages/manager/apps/dedicated/client/app/ip/ip/ip-ip.html:72]: Okay Anoop

**Review by y4nnL — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.routes.js (hunk 1 above)
Comment (mohammed-zahaf) [reply packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.routes.js:12]: Yep, we can you have to create Jira ticket to not forget it

**Review by mohammed-zahaf — APPROVED** [review]

**Pull request merged**